src/renderer/marketplace/keyfi/checkout/keyfi-pay-container.jsx

```diff
@@ -0,0 +1,91 @@
+import BN from 'bignumber.js';
+import { connect } from 'react-redux';
+import { push } from 'connected-react-router';
+import { withStyles } from '@material-ui/styles';
+import { featureIsEnabled } from 'common/feature-flags';
+import { getWallet } from 'common/wallet/selectors';
+import { kycSelectors } from 'common/kyc';
+import { pricesSelectors } from 'common/prices';
+import { marketplaceSelectors } from 'common/marketplace';
+import { ordersOperations } from 'common/marketplace/orders';
+import { MarketplaceKeyFiComponent } from '../common/marketplace-keyfi-component';
+import { identitySelectors } from 'common/identity';
+
+const styles = theme => ({});
+
+class KeyFiPayContainer extends MarketplaceKeyFiComponent {
+	async componentDidMount() {
+		await this.loadRelyingParty({ rp: this.props.vendorId, authenticated: true });
+		await this.createOrder();
+	}
+
+	priceInKEY = priceUSD => {
+		return new BN(priceUSD * 0.8).dividedBy(this.props.keyRate).toString();
+	};
+
+	priceInETH = priceUSD => {
+		return new BN(priceUSD).dividedBy(this.props.ethRate).toString();
+	};
+
+	async createOrder() {
+		const { product, vendorId, vendor, cryptoCurrency } = this.props;
+		const application = this.getLastApplication();
+		const price =
+			cryptoCurrency === 'KEY'
```

Comment:
Will it work wit KI?

Reply:
No it wouldn't, replaced it with primaryToken.